### PR TITLE
Milestone 2: Components redux store

### PR DESCRIPTION
**Pull request by imahnama**
## In this Milestone we: 
- Prepare the directory structure
- Implement the App component
- Implement the BooksList component
- Implement the Book component
- Implement the BooksForm component
- Define the Redux actions and reducers
- Initialize the Redux store

### CODE REVIEW

**Review by fatymahmed — APPROVED** [review]
## Hi @imahnama @mbabaliiryn 👋
You have done a great job in this milestone🎉🎉🎉  
## Status: Approved ✔️
**Kindly check my comment below before merging the PR**
_You may go ahead and merge your PR_

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ACTIONTYPES from '../constant/ActionTypes';
+
+export const createBook = book => ({
+  type: ACTIONTYPES.CREATE_BOOK,
+  book,
+});
+
+export const removeBook = id => ({
+  type: ACTIONTYPES.REMOVE_BOOK,
+  id,
```
Comment (fatymahmed) [inline src/actions/index.js:10]: this should also be expecting a `book` as an argument as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReduxActions/ReducxActions.png)
**please make this change before merging**

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      {' '}
+      {book.category}
+      {' '}
+    </td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.shape({}).isRequired,
+};
```
Comment (fatymahmed) [inline src/components/Book.js:27]: **[OPTIONAL]** - This is marked as optional, therefore you may decide to do make this change or not
You could use `object` for typechecking as mentioned in the react [documentation](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes)

`book: PropTypes.object.isRequired`

Comment (imahnama) [reply src/components/Book.js:27]: Noted thanks for the suggestion

**Pull request merged**